### PR TITLE
Update free-courses-ml.md

### PR DESCRIPTION
**Pull request by RijoKsd**
Ordered playlist in alphabetic order
Added a new Malayalam playlist of
* Android
* Angular
* Bootstrap
* C
* CSS
* CPP
* C#
* Flask
* Flutter
* Game Development
* Git
* HTML
* JAVA
* JavaScript
* Linux
* Laravel
* MYSQL
* Next JS
* Node JS
* PHP

## What does this PR do?

Added resources and improved repo

### Why is this valuable (or not)?

This open-source contribution is valuable because it allows Malayali students to learn coding in their native language, Malayalam. By making coding more accessible to students in their native language, this open-source contribution can help to increase access to coding resources and potentially open up a new world of opportunities for Malayali students.
NB: Malayalam is the native language of Kerala. Kerala is situated in INDIA

### For book lists, is it a book? For course lists, is it a course? etc.

This is an online course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ml.md [hunk 1]
```diff
@@ -1,22 +1,31 @@
 ### Index
 
+* [Index](#index)
```
Comment (eshellman) [inline courses/free-courses-ml.md:3]: remove this

File: courses/free-courses-ml.md [hunk 2]
```diff
@@ -37,11 +46,20 @@
...
 ### <a id="cpp"></a>C++
 
 * [C++ Malayalam Tutorial Full](https://youtube.com/playlist?list=PL1JrLEBAapUXVcHV3rO-g-N4gBHcBe2c-) - Yes Tech Media
+* [C++ Course Malayalam](https://www.youtube.com/playlist?list=PLefF9wgiOa4VZ4RuSArY4mNO8Roz79cUU) - Nikin Sk
+
+
+### <a id="csharp"></a>C\#
+
+* [Learn C# For Beginners in Malayalam](https://www.youtube.com/playlist?list=PLQzHWfiB9fWfGQWWU7RhIAp1DH3rzSUc1) - Learn Programming In Malayalam
+* [C# (C Sharp) Malayalam Tutorial](https://www.youtube.com/playlist?list=PL1JrLEBAapUW5ijzyEqYxRawB73rsjHW9) - Yes Tech Media
```
Comment (eshellman) [inline courses/free-courses-ml.md:61]: L > C

File: courses/free-courses-ml.md [hunk 3]
```diff
@@ -54,15 +72,29 @@
 * [Data Structure Challenge](https://youtube.com/playlist?list=PLY-ecO2csVHc5kajCd3fHU6MhkTABkRh9) - Brototype Malayalam
 
 
+### Docker
+
+* [Docker Tutorial](https://www.youtube.com/playlist?list=PLmFap_OoBJh7Z5tTSkeXciuIMZnKdJ1uu) - Tech Trivandrum
```
Comment (eshellman) [inline courses/free-courses-ml.md:77]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

File: courses/free-courses-ml.md [hunk 4]
```diff
@@ -54,15 +72,29 @@
...
+
+
 ### Flutter
 
 * [Flutter Malayalam Tutorial Series](https://youtube.com/playlist?list=PLY-ecO2csVHcUlBVvIMAa3dbja12TFJiN) - Brototype Malayalam
+* [Flutter Malayalam Tutorial](https://www.youtube.com/watch?v=l0f7olpevO0&list=PL1JrLEBAapUWv_JSolgE5pdcDKWw80wKA) - Yes Tech Media
+* [Flutter Beginner Tutorials](https://www.youtube.com/playlist?list=PLr11YFCnWCCMQYU8z3Gol2gVA2nBtoKVo) - Mallu Developer
```
Comment (eshellman) [inline courses/free-courses-ml.md:84]: B < M

File: courses/free-courses-ml.md [hunk 5]
```diff
@@ -54,15 +72,29 @@
...
+
+
+### Game Development
+
+* [Game Development Tutorial Malayalam | Godot](https://www.youtube.com/playlist?list=PLcP4K64TKma33Vz9AhPvJpVJuO5jRbcyw) - Spotix
+* [Game development challenge tutorial series](https://www.youtube.com/playlist?list=PLY-ecO2csVHegnXkm4aLAZTRF_3nTMJMW) - Brototype Malayalam
```
Comment (eshellman) [inline courses/free-courses-ml.md:90]: T > c

File: courses/free-courses-ml.md [hunk 6]
```diff
@@ -71,23 +103,44 @@
...
 #### Bootstrap
 
 * [BOOTSTRAP Malayalam Tutorial](https://youtube.com/playlist?list=PL1JrLEBAapUWqs_HbcYngAOmpPbiccqNy) - Yes Tech Media
+* [Bootstrap Malayalam Tutorials](https://www.youtube.com/playlist?list=PLDavEIls6jrsC9EEox1WWvOlktDJcEUo6) - Malayalam Tutorials
 
 
 ### Java
 
 * [Java Programming Malayalam Tutorial](https://youtube.com/playlist?list=PL1JrLEBAapUXndScHzvMaSVOspebvFOsH) - Yes Tech Media
+* [Java](https://www.youtube.com/playlist?list=PLY-ecO2csVHeKaBI7lAM1jbIPU8K6fUxY) - Brototype
```
Comment (eshellman) [inline courses/free-courses-ml.md:118]: 'java' < 'java p'

File: courses/free-courses-ml.md [hunk 7]
```diff
@@ -102,6 +155,7 @@
 * [React JS - Malayalam Tutorial - by Yes Tech Media](https://youtube.com/playlist?list=PL1JrLEBAapUX44d5zzl0hf7vX7caSCidT) - Yes Tech Media
 * [React js malayalam tutorial](https://youtube.com/playlist?list=PL4KwFGqvN4nuKJ3bJSP-LcchabWJtPtE8) - Debug Media
 * [React js malayalam tutorial](https://youtube.com/playlist?list=PLBKJGko2JEdG8FUKu2hUF6K7irvbT8hIM) - Tintu Vlogger
+* [React in malayalam](https://www.youtube.com/playlist?list=PLCOzHVsG8mkrePHYn73VBEK4dqwa4W7LU) - Web Diar
```
Comment (eshellman) [inline courses/free-courses-ml.md:158]: i < j

File: courses/free-courses-ml.md [hunk 8]
```diff
@@ -110,22 +164,39 @@
 * [Kotlin Malayalam | Kotlin programming Tutorials for android malayalam](https://www.youtube.com/playlist?list=PLaP7lUdqAGYPpEutAk6o1jKC08Rc5xMXs) - CODEAVIAL (YouTube)
 
 
+### Linux
+
+* [Linux Malayalam](https://www.youtube.com/playlist?list=PLmFap_OoBJh6ZIbXANxdBYAlKCCAngX5o) - Tech Trivandrum
+* [Learn Linux Malayalam](https://www.youtube.com/playlist?list=PLQzHWfiB9fWcImNxiu-wgnurqLM3fKNDY) - Learn Programming In Malayalam
```
Comment (eshellman) [inline courses/free-courses-ml.md:170]: Le < Li

File: courses/free-courses-ml.md [hunk 9]
```diff
@@ -110,22 +164,39 @@
...
+
+
 ### MySQL
 
 * [MySQL Malayalam Tutorials for beginners](https://www.youtube.com/playlist?list=PLQzHWfiB9fWcxc10XNwm7CeOEMrjVr8a9) - Learn Programming In Malayalam
+* [Mysql Malayalam tutorial series](https://www.youtube.com/playlist?list=PLBKJGko2JEdFKFGGVaveL5IOxxSNq3eBS) - Tintu Vlogger
```
Comment (eshellman) [inline courses/free-courses-ml.md:176]: tutorial < tutorials

File: courses/free-courses-ml.md [hunk 10]
```diff
@@ -110,22 +164,39 @@
...
+* [Laravel Tutorial for Beginners in Malayalam](https://www.youtube.com/playlist?list=PLbasZIkCgHJFQ2MxkwhokpGPyL4MKVAQt) - Code Band
+* [Laravel Tutorials in malayalam](https://www.youtube.com/playlist?list=PL-Z1WBeTYPOpqIe09MwnH62VwOLAj2tJz) - Aks Programming
 
 
 ### Python
 
 * [Python](https://www.youtube.com/playlist?list=PLwe8bwPXjlLveEHvTbKMXJOPkFdXnu4xi) - Learn CSE Malayalam
 * [Python Programming Malayalam Tutorial](https://www.youtube.com/watch?v=ihnWXGPxNEk) - Yes Tech Media
 * [Python Tutorial Malayalam \| Best Malayalam Python Programming Tutorial](https://youtube.com/playlist?list=PLY-ecO2csVHfbpOmWamlb8Mujjdnl1jks) - Brototype Malayalam 
+* [Python Django malayalam](https://www.youtube.com/playlist?list=PLFUS7KFPLkNYStmg2qIUGctQ2OOO0Mw63) - DotPy Media
```
Comment (eshellman) [inline courses/free-courses-ml.md:199]: D < P

**Review by eshellman — CHANGES_REQUESTED** [review]
Please make edits on this PR (don't close it - that wipes out the comments)

**Comment by RijoKsd** [comment]
🎉 Added playlist in alphabetic order

File: courses/free-courses-ml.md [hunk 11]
```diff
@@ -37,13 +45,22 @@
 * [C Programming](https://www.youtube.com/playlist?list=PLwe8bwPXjlLtUxDFwafx_XvheCUmrP2cM) - Learn CSE Malayalam (YouTube)
 * [C programming for beginners in malayalam quarantine learning programming malayalam](https://youtube.com/playlist?list=PLBKJGko2JEdEVpp5vyyfKYdv3r2GC777i) - Tintu Vlogger
 * [C programming malayalam tutorial](https://youtube.com/playlist?list=PL1JrLEBAapUXh0dSzCk3dvQtGmjT9fBBj) - Yes Tech Media
+* [C programming](https://www.youtube.com/playlist?list=PLY-ecO2csVHeKaBI7lAM1jbIPU8K6fUxY) - Brototype malayalam
```
Comment (eshellman) [inline courses/free-courses-ml.md:48]: needs to move up to the other one with the same title

**Review by eshellman — CHANGES_REQUESTED** [review]
Almost!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**